File: Ruby/fibonacci.rb
a = 0
puts a
b = 1
puts b

while true
  c = a + b
  
  if c > 18446744073709551615
    puts "Overflowed!"
    break
  end

  cstr = "%u" % c
  puts cstr

  a = b
  b = c
end

File: PHP/fibonacci.php
<?php
$maximum = 18446744073709551615;
$a = 0;
echo "$a\n";
$b = 1;
echo "$b\n";

while(($a + $b) < $maximum) {
    $c = $a + $b;
    $cstr = sprintf('%lu', $c);    
    echo "$cstr\n";
    $a = $b;
    $b = $c;
}

echo "Overflowed!\n";
?>
